packaging/packager_deb.sh

```diff
@@ -9,62 +9,72 @@ readonly distro_root="${ROOT:-$HOME}"
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
 # python3
-apt-get -y install python3 python3-coverage
-# python-imaging
-apt-get -y install python3-pil
+apt-get -y install python3 python3-coverage python3-venv
 # pip dependencies (for dependencies not available as DEB)
 apt-get -y install gcc libx11-dev libxtst-dev python3-dev libpng-dev python3-pip
+# create a venv to use that as the python env
+rm -rf docker_venv
+python3 -m venv docker_venv
+source docker_venv/bin/activate
+pip3 install --upgrade pip
+# setup tools
+pip3 install setuptools setuptools-rust
+# python-imaging
+apt-get -y install python3-pil
+pip3 install pillow==10.4.0
+# upgrade pip
 pip3 install --upgrade pip
-# contour, template, feature, cascade, text matching
+
+echo "contour, template, feature, cascade, text matching"
 apt-get -y install python3-numpy
-if [[ $distro_version == "xenial" ]]; then
-    export DISABLE_OPENCV=1
-else
-    apt-get -y install python3-opencv
-fi
-# text matching
-if [[ $distro_version == "xenial" ]]; then
-    export DISABLE_OCR=1
-else
-    apt-get -y install tesseract-ocr libtesseract-dev
-    apt-get -y install g++ pkg-config
-    pip3 install pytesseract==0.3.4 tesserocr==2.5.1
-fi
-# deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
-# screen controlling
-pip3 install autopy==4.0.0
+pip3 install numpy
+pip3 install opencv-python
+apt-get -y install tesseract-ocr libtesseract-dev
+apt-get -y install g++ pkg-config
+pip3 install pytesseract==0.3.10 tesserocr==2.7.0
+
+echo "deep learning"
+pip3 install torch==2.2.0 torchvision==0.17.0
+
+echo "screen controlling"
+export DISABLE_AUTOPY=1
 # TODO: vncdotool doesn't control its Twisted which doesn't control its "incremental" dependency
 pip3 install incremental==22.10.0
 pip3 install vncdotool==0.12.0
 apt-get -y install xdotool x11-apps imagemagick
 apt-get -y install python3-tk scrot
-# TODO: install PyScreeze separately to replace the one PyAutoGUI will install which is incompatible with the current Pillow version 
 pip3 install pyscreeze==0.1.28
-pip3 install pyautogui==0.9.53
+apt-get -y install gnome-screenshot
+pip3 install pyautogui==0.9.54
 apt-get -y install x11vnc
 
-# deb packaging and installing of current guibot source
+echo "------------- deb packaging and installing of current guibot source -------------"
 apt-get -y install dh-make dh-python debhelper python3-all devscripts
 NAME=$(sed -n 's/^Package:[ \t]*//p' "$distro_root/guibot/packaging/debian/control")
 CHANGELOG_REVS=($(sed -n -e 's/^guibot[ \t]*(\([0-9]*.[0-9]*\)-[0-9]*).*/\1/p' "$distro_root/guibot/packaging/debian/changelog"))
 VERSION=${CHANGELOG_REVS[0]}
+
 cp -r "$distro_root/guibot" "$distro_root/$NAME-$VERSION"
 cd "$distro_root/$NAME-$VERSION/packaging"
+
+echo "------------- debuild -------------"
 debuild --no-tgz-check --no-lintian -i -us -uc -b
 cp ../${NAME}_${VERSION}*.deb "$distro_root/guibot"
+
+echo "------------- install guibot -------------"
```

Comment:
Is it good to keep the various packager scripts in sync as much as possible - this means either adding such echos in the rpm and other relatable scripts or removing them everywhere.

Reply:
Ok, makes sense